tutorial/dashbio.py

```diff
@@ -0,0 +1,242 @@
+import dash_core_components as dcc
+import dash_html_components as html
+from tutorial import styles
+
+import dash_bio
+from tutorial.utils.dashbio_docs import generate_docs
+
+dashbio_library_heading = dcc.Markdown('''
+    # Dash Bio
+
+    Dash is a web application framework that provides pure Python abstraction
+    around HTML, CSS, and JavaScript.
+
+    Dash Bio is a suite of bioinformatics components that make it simpler to
+    analyze and visualize bioinformatics data and interact with it in a Dash
```

Comment:
```suggestion
    analyze and visualize bioinformatics data and interact with them in a Dash
```
In general, *data* is plural...

Reply:
@mkcor Good catch :)